Slayer Assistant Plugin

LeeOkeefe opened the pull request:
Plugin for searching for slayer tasks and showing information related to each task.

- Monster locations
- Items required/recommended
- Monster attack styles and attributes
- Slayer masters that can assign the task

Nightfirecat commented:
The "slayer" config group is already in use by the core slayer plugin, use a unique string here. Alternatively, given your plugin doesn't use any configuration, you could simply remove the config class entirely.

LeeOkeefe commented:
> The "slayer" config group is already in use by the core slayer plugin, use a unique string here. Alternatively, given your plugin doesn't use any configuration, you could simply remove the config class entirely.

Oops! I've removed the config file and updated the commit hash.
